geth-dev-node/genesis.json

```diff
@@ -0,0 +1,18 @@
+{
+  "config": {
+        "chainId": 10000,
+        "homesteadBlock": 0,
+        "eip155Block": 0,
+        "eip158Block": 0
+    },
+  "alloc"      : {},
+  "coinbase"   : "0x0000000000000000000000000000000000000000",
+  "difficulty" : "0x400",
```

Comment:
Were you able to figure out how to make Geth have shorter block times?  Nothing in this file stands out to me as such, but maybe I just don't know what to look for.

What does `difficulty` represent?

Reply:
I checked with @epheph about this, but he said you have to use proof of authority, become the authority, and then run a miner as the authority at a specific interval.  However, he has not tried this before.

Reply:
I believe difficulty just represents the block difficulty.

Reply:
difficulty represents initial block difficulty. Initial blocks are mined VERY quickly, but they converge toward normal block times, and I don't believe there's an override for target block time.

Another solution is to come up with a system that doesn't mine 0-TX blocks, you'd keep that difficulty low for a long time.